deploy/core/node_modules/shelljs/shell.js

```diff
@@ -389,8 +389,8 @@ function _rm(options, files) {
     // If here, path exists
 
     var stats = fs.statSync(file);
-    // Remove simple file
-    if (stats.isFile()) {
+    // Remove simple file (or Electron package files which 'fs' has been patched by Electron to treat as directories)
```

Comment:
It seems like it's not true with that PR merged. What's weird is that PR was merged on the same date as we last bumped Electron in LT. But we bumped to 0.34.5 and that PR seems to be part of 0.35.3. @sbauer322 might test upgrading to a newer version.

I'll try to do so myself for the changes in this PR too. Given that everyone's going to need to either download the next version or build it themselves from source, upgrading Electron now has no marginal impact. Not having to handle the ASAR workaround should simplify this greatly. Maybe we could even get rid of some other Node.js dependencies.